README.md

```diff
@@ -186,7 +186,8 @@ scrape_configs:
 | web.telemetry-path                  | REDIS_EXPORTER_WEB_TELEMETRY_PATH                | Path under which to expose metrics, defaults to `/metrics`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| redis-only-metrics                  | REDIS_EXPORTER_REDIS_ONLY_METRICS                | Whether to also export go runtime metrics, defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| redis-only-metrics                  | REDIS_EXPORTER_REDIS_ONLY_METRICS                | Whether to export only Redis metrics (omit Go process+runtime metrics), defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| include-go-runtime-metrics          | REDIS_INCLUDE_GO_RUNTIME_METRICS                 | Whether to include Go runtime metrics, defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
REDIS_INCLUDE_GO_RUNTIME_METRICS -> `REDIS_EXPORTER_INCLUDE_GO_RUNTIME_METRICS`

Reply:
Fixed, thank you 👍